### PR TITLE
 PLAT-139288: Change drop-shadow to box-shadow

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Shadow effect make performance issue on TV board.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Replace drop-shadow with box-shadow

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Tooltip component should use drop-shadow due to it's shape.

There was two regression about Input / Inputfield
- Input Field does not have shadow when focus (=>I added "box-shadow: none;")
- Input field have "filter+_: brightness" when disabled, it should be unset when focus (I added "filter: unset;")

### Links
[//]: # (Related issues, references)
PLAT-149288

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]
:lgtm:

File: styles/color-mixins.less [hunk 1]
```diff
@@ -80,7 +80,7 @@
 	opacity: 1;
 	.sand-bg-colors({
 		background-color: @sand-spotlight-bg-color;
-		filter+: drop-shadow(@sand-spotlight-focus-shadow);
+		box-shadow: @sand-spotlight-focus-shadow;
```
Comment (MikyungKim) [inline styles/color-mixins.less:83]: I think we need to add `filter` to https://github.com/enactjs/sandstone/pull/956/files#diff-6fa658458e34133889fd42ba25feceb3b5bdcc8312532446734665242427273cR159.
And change https://github.com/enactjs/sandstone/pull/956/files#diff-6fa658458e34133889fd42ba25feceb3b5bdcc8312532446734665242427273cR71 to `box-shadow`.

File: ImageItem/ImageItem.module.less [hunk 2]
```diff
@@ -193,7 +193,7 @@
 			&.vertical {
 				&.fullImage .image {
 					transform: @sand-imageitem-vertical-focus-image-transform;
-					filter: drop-shadow(@sand-imageitem-focus-bg-shadow);
+					box-shadow: @sand-imageitem-focus-bg-shadow;
```
Comment (MikyungKim) [inline ImageItem/ImageItem.module.less:196]: Please change https://github.com/enactjs/sandstone/pull/956/files#diff-c3fea8bd3ecfb212df7c3d670b4e4176fc62743c557cf1d72801c677481b23b3R74-R75 too.

**Comment by MikyungKim** [comment]
Let's ask GUI person's opinion.

**Review by MikyungKim — APPROVED** [review]
:shipit:

**Pull request merged**